clusterloader2/pkg/framework/framework.go

```diff
@@ -92,6 +92,12 @@ func (f *Framework) DeleteAutomanagedNamespaces() error {
 			return err
 		}
 	}
+	for i := 1; i <= f.automanagedNamespaceCount; i++ {
```

Comment:
I would suggest "deleting and waiting" each namespace in a separate go routine. Otherwise it is unfair for earlier namespaces which get less time to finish compared to later namespaces.

Reply:
Rewrited to go routines.